gradle/libs.versions.toml

```diff
@@ -96,6 +98,9 @@ spring-data-commons = { module = "org.springframework.data:spring-data-commons",
 
 micronaut-azure-cosmos = { module = "io.micronaut.azure:micronaut-azure-cosmos", version.ref = "micronaut-azure" }
 
+# Reactor Netty Dependencies
```

Comment:
Should be removed when https://github.com/micronaut-projects/micronaut-r2dbc/pull/669 is done and r2dbc 5.6.1 is released